dev-duckies-sm-proj/src/pages/public-feed/public-feed.jsx

```diff
@@ -1,10 +1,35 @@
-export default function PublicFeed() {
+import Header from "../../app-wide-components/Navbar/MobileNav.jsx";
+import Post from "../../app-wide-components/post.jsx";
+import "../../index.css";
+
+function PublicFeed() {
   return (
-    <section className="section">
-      <h1 className="title">Feed</h1>
-      <h2 className="subtitle">
-        A home/feed/explore page for all posts by all users.
-      </h2>
-    </section>
-  )
-}
\ No newline at end of file
+    <div className="App">
+      <Header />
+      <div className="columns">
+        <div className="column is-12">
+          <Post
+            user="Tommy"
```

Comment:
Update so the props.username is displayed
```suggestion
            username="Tommy"
```